packages/core/src/asciidoctor-core-api.js

```diff
@@ -1857,6 +1857,41 @@ Document.prototype.hasHeader = function () {
   return this['$header?']()
 }
 
+/**
+ * Replay attribute assignments at the block level
+ *
+ * @param {Object} blockAttributes - A JSON of attributes
```

Comment:
This might be misleading.  The only real use case for this is to playback the actual document attributes, which I would expect to be an Opal hash rather than JSON.

Reply:
Actually if you are using the JavaScript API, `block.getAttributes()` will also return JSON.
`Opal.hash` should not leak in the Asciidoctor.js API.